protobuf/vm.proto

```diff
@@ -0,0 +1,20 @@
+
+syntax = "proto3";
+
+package protobuf;
+
+option go_package="github.com/kubearmor-client/protobuf";
```

Comment:
```suggestion
option go_package="github.com/kubearmor/kubearmor-client/protobuf";
```

Also I wonder if we should keep this protobuf inside the `/vm` directory since this is related to vm subcommand only! What do you think?

Then it would be 
```suggestion
option go_package="github.com/kubearmor/kubearmor-client/vm/protobuf";
```

Reply:
Handled